keanu-python/.gitignore

```diff
@@ -4,4 +4,5 @@ Pipfile.lock
 __pycache__/
 dist/
 keanu/classpath
+nd4j/nd4j/classpath
```

Comment:
Because it needs its own `setup.py` I had to make a directory (because keanu-python's one is in this dir). Then `setup.py` looks in the directory of the package name to know what to package up.

Reply:
👍